docs/src/main/sphinx/sql/create-role.rst

```diff
@@ -9,17 +9,21 @@ Synopsis
 
     CREATE ROLE role_name
     [ WITH ADMIN ( user | USER user | ROLE role | CURRENT_USER | CURRENT_ROLE ) ]
+    [ IN catalog ]
 
 Description
 -----------
 
-``CREATE ROLE`` creates the specified role in the current catalog.
+``CREATE ROLE`` creates the specified role.
 
 The optional ``WITH ADMIN`` clause causes the role to be created with
 the specified user as a role admin. A role admin has permission to drop
 or grant a role. If the optional ``WITH ADMIN`` clause is not
 specified, the role is created with current user as admin.
 
+The optional ``IN catalog`` clause creates the role in a catalog as opposed
+to a system role.
```

Comment:
I think we need to explain somewhere what is a difference between catalog role vs system role. Like explain what this actually means for the user and plugin developer.

Reply:
These docs are for end users, so it definitely shouldn't go here.  We don't have any documentation on writing access control implementations.